docs/protocol/network/overview.md

```diff
@@ -0,0 +1,52 @@
+# Witnet Network Protocol
+
+Witnet network protocol is inspired in other blockchain network protocols such as Bitcoin, Ethereum, Exonum, Mimblewimble Grin and Rchain. Additionally, other blockchains such as Grin and Exonum have been taken into consideration as their reference implementations are coded in Rust.
```

Comment:
Witnet network protocol is inspired by other blockchain network protocols such as Bitcoin, Ethereum, Exonum, Mimblewimble Grin and Rchain. Some of the aforementioned blockchains architecture have been taken into consideration as their reference implementations are also coded in Rust.